tracer/src/Datadog.Trace/Debugger/ExceptionAutoInstrumentation/ExceptionTrackManager.cs

```diff
@@ -52,7 +52,8 @@ private ExceptionTrackManager(ExceptionReplaySettings settings)
             _cachedInvalidatedCases = new();
 
             _exceptionProcessorTask = Task.Factory.StartNew(
-                                               async () => await StartExceptionProcessingAsync(_cts.Token).ConfigureAwait(false), TaskCreationOptions.LongRunning)
+                                               () => StartExceptionProcessingAsync(),
+                                               TaskCreationOptions.LongRunning)
```

Comment:
You probably need to make sure you're using the default scheduler if you're using `StartNew` instead of `Task.Run()`

Reply:
Fixed in dd64f6e89a16c1969abd9c1deb54d70d0f8c89c4